deployments/env_setup/kube-cluster/aws-cli

```diff
@@ -0,0 +1,6 @@
+aws ec2 describe-instances \
+--filters \
+"Name=instance-state-name,Values=running" \
+"Name=tag:Name,Values=ym-250-4" \
+--query 'Reservations[*].Instances[*].[PrivateIpAddress]' \
+--output text
```

Comment:
@yanmo96 What is this file for?

Reply:
This is used to get list of aws ec2 we created with name as filter, so we can then put them into the ansible host much faster than click on the webpage one by one

Reply:
@yanmo96 Please make a comment in front of the file to describe what's the purpose of this file.